BootloaderCommonPkg/Library/SmbiosInitLib/SmbiosTables.h

```diff
@@ -0,0 +1,161 @@
+/** @file
+
+  Copyright (c) 2019, Intel Corporation. All rights reserved.<BR>
+  This program and the accompanying materials
+  are licensed and made available under the terms and conditions of the BSD License
+  which accompanies this distribution.  The full text of the license may be found at
+  http://opensource.org/licenses/bsd-license.php.
+
+  THE PROGRAM IS DISTRIBUTED UNDER THE BSD LICENSE ON AN "AS IS" BASIS,
+  WITHOUT WARRANTIES OR REPRESENTATIONS OF ANY KIND, EITHER EXPRESS OR IMPLIED.
+
+**/
+
+#ifndef __SMBIOS_TABLES_H__
+#define __SMBIOS_TABLES_H__
+
+#include <Library/BaseLib.h>
+#include <IndustryStandard/SmBios.h>
+
+#define TO_BE_FILLED                                0
+
+#define SMBIOS_MISC_STRING_NULL                     0
+#define SMBIOS_MISC_STRING_1                        1
+#define SMBIOS_MISC_STRING_2                        2
```

Comment:
How about using SMBIOS_STRING_INDEX_xxx instead of SMBIOS_MISC_STRING_xxx?

Reply:
Done.